app/modules/ads/module.js

```diff
@@ -51,17 +51,25 @@ class Ads {
   }
 
   callExternals() {
+    const { context } = window.Wikia.adEngine;
     const { bidders } = window.Wikia.adBidders;
-    const { geoEdge, krux, moatYi } = window.Wikia.adServices;
+    const { geoEdge, krux, moatYi, nielsen } = window.Wikia.adServices;
 
     biddersDelay.resetPromise();
     bidders.requestBids({
       responseListener: biddersDelay.markAsReady,
     });
 
+    const targeting = context.get('targeting');
+
     geoEdge.call();
     krux.call();
     moatYi.call();
+    nielsen.call({
```

Comment:
Is it fine/desired to be called on each pv?

Reply:
Yes, cause we want to send new parameters for each page.